test/common/router/config_impl_test.cc

```diff
@@ -6265,7 +6273,7 @@ name: AllRedirects
         redirect: { host_redirect: "new.lyft.com", prefix_rewrite: "/new/prefix" , https_redirect: "true", strip_query: "true" }
   )EOF";
 
-  TestConfigImpl config(parseRouteConfigurationFromV2Yaml(RouteDynPathRedirect), factory_context_,
+  TestConfigImpl config(parseRouteConfigurationFromV3Yaml(RouteDynPathRedirect), factory_context_,
```

Comment:
Are these all v3? I'm guessing some haven't been updated. Doing the parse for v3 with `do_boosting = false` would probably show.

Reply:
You are correct, I've updated the loading function name